src/app/zap-templates/partials/header.zapt

```diff
@@ -1,6 +1,6 @@
 /*
  *
- *    Copyright (c) 2020 Project CHIP Authors
+ *    Copyright (c) 2021 Project CHIP Authors
```

Comment:
Mind doing a separate PR for that ? 

This change will affect all generated files and one may found confusing to see this change in various file for a completely different change.

So I think doing a separate PR with this change and which update all generated files would be better than sneaking this change here :)

Reply:
I think so, a seperate PR is better.